---
title: "Paper_notes"
author: "Narad Mishra"
date: "October 1, 2017"
output: html_document
---


## High density synthetic oligonucleotide arrays

This paper written by Robert J. Lipshutz, et.al., is a discreption of DNA probe arrays developed by a company Affymetrix. They have used the simple principle of recognition and hybridization ability of DNA sequnce to develop a new experimental tools designed to collect and analyse vast amounts of genetic and cellular information. The high-density DNA probe arrays developed at Affymatrix used two techniques, photolithography and solid-phase DNA synthesis. This high density DNA probe arrays are used for gene expression monitoring, which then is used for genotypic analysis. The paper describes on how this Oligonucleotide arryas for gene expression monitoring are designed, how are they analyzed and new tools and softwares that are currently being developed by Affymetrix.

---

## Expression profiling; best practices for data generation and interpretation in clinical trials

This papar talks about the importance of Microarrays for mRNA transcript expression level detection and analysis. These microarrays are being used increasinly in clinical trials. Due to regulations in clinical trials, microarry technology and the data generated must have clear set standards for its use. This paper tries to find the best practice for data generation and its interpretation, so it can be used in clinical trials. For this, groups like *Microarray Gene Expression Data* (MGED) society, *Minimum Information About A Microarray Experiment* (MIAME), and the MAGE-ML mark-up language are step closer to aquaring unity and standarizing the procedure on how to approach to the obtaining of and interpretation of genomic data. The paper tries to show the key differences between Affymetric arrays, spotted cDNA, and spotted oligonucleotide. The paper also describes how and where could there be possible variation in the data and recommends the best practice to experimental designs. 