---
title: "Paper Summary"
author: "Narad Mishra"
output: html_document 
---

## Watson & Crick, 1953
Watson, J. D., and F. H. Crick. *"Molecular Structure of Nucleic Acids; a Structure for Deoxyribose Nucleic Acid."* Nature 171, no. 4356 (April 25, 1953): 737-38   

  In this paper, Watson & Creek wish to suggest the structure of Deoxyribose Nucleic Acid (DNA), which they have refered to as a salt of Deoxyribose Nucleic Acid. A structure for DNA was previously proposed by Pauling and Corey, that pictured the DNA as having three inter-twined chains with the phosphates near the fibre axis, and the bases on the outside. This structure didn't have explanation on how the chains having all negatively charged stayed that close without being repelled. Their proposed structure of DNA consist of two helical chains each coiled around a fibre axis. The two chains are related by a dyad perpendicular to the fibre axis. Each chain consist of bases on te inside and the phosphates on the outside. In this structure, the sugars are being roughly perpendicular to the attached base. The bonding between the bases of a chain to the base of other chain makes the two chains to stick together. Purine base of a chain is hydrogen bonded with pyrimidine base of the other chain. This bonding are perpendicular to the fibre axis. This bonding is said to be in a way such that purine position 1 bonds to pyrimidine position 1, and also position 6 of purine bonds to position 6 of pyrimidine. Due to this specific base pairing, it is possible to come up with the sequence of a chain given the sequence of the other chain. Previously available x-ray data was insufficient to prove their proposed structure, so they wanted the work to be regarded as unproved until the structure is checked against more exact results. They finally specified that their proposed specific base pairing suggest for possible copying mechanism for the genetic material. The paper acknowlegdes the help and support they received from different individuals and papers.
  
---


## High density synthetic oligonucleotide arrays

  This paper written by Robert J. Lipshutz, et.al., is a discreption of DNA probe arrays developed by a company Affymetrix. They have used the simple principle of recognition and hybridization ability of DNA sequnce to develop a new experimental tools designed to collect and analyse vast amounts of genetic and cellular information. The high-density DNA probe arrays developed at Affymatrix used two techniques, photolithography and solid-phase DNA synthesis. This high density DNA probe arrays are used for gene expression monitoring, which then is used for genotypic analysis. The paper describes on how this Oligonucleotide arryas for gene expression monitoring are designed, how are they analyzed and new tools and softwares that are currently being developed by Affymetrix.

---

## Expression profiling; best practices for data generation and interpretation in clinical trials

  This papar talks about the importance of Microarrays for mRNA transcript expression level detection and analysis. These microarrays are being used increasinly in clinical trials. Due to regulations in clinical trials, microarry technology and the data generated must have clear set standards for its use. This paper tries to find the best practice for data generation and its interpretation, so it can be used in clinical trials. For this, groups like *Microarray Gene Expression Data* (MGED) society, *Minimum Information About A Microarray Experiment* (MIAME), and the MAGE-ML mark-up language are step closer to aquaring unity and standarizing the procedure on how to approach to the obtaining of and interpretation of genomic data. The paper tries to show the key differences between Affymetric arrays, spotted cDNA, and spotted oligonucleotide. The paper also describes how and where could there be possible variation in the data and recommends the best practice to experimental designs. 
